content/custom_resources/lwrp_to_custom_resources.md

```diff
@@ -0,0 +1,123 @@
++++
+title = "Migrating from LWRPs to Custom Resources"
+gh_repo = "chef-web-docs"
+
+[menu]
+  [menu.infra]
+    title = "Migrating from LWRPs"
+    identifier = "chef_infra/cookbook_reference/custom_resources/lwrp"
+    parent = "chef_infra/cookbook_reference/custom_resources"
+    weight = 50
++++
+
+## Overview
+
+It is no longer recommended to write resources in the __Light Weight Resource Provider (LWRP)__ format.
+
+This guide describes how to migrate from an existing LWRP to a Custom Resource.
+
+If you are creating a Custom Resource from scratch please see the [Custom Resource Getting Started Guide](getting-started.md) instead.
+
+## Convert files to Custom Resources Layout
+
+LWRPs are a pair of library files one provider, one resource.
```

Comment:
Suggest: LWRPs consist of two library files -- a resource and a provider for that resource.